app/src/main/res/values/styles.xml

```diff
@@ -637,6 +637,7 @@
     <item name="android:textSize">20sp</item>
   </style>
 
+
```

Comment:
Is the double newline intentional?

Reply:
No, this probably came about from resolving conflicts.

Reply:
Fixed